tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
 
 If your Apache Kafka service is in a GCP VPC that has no internet access, take the following steps:
 
-1. [Set up a VPC peering connection](/tidb-cloud/set-up-vpc-peering-connections.md) between the VPC of the Apache Kafka service and your TiDB cluster. 
-2. Modify the ingress firewall rules of the VPC where Apache Kafka is located. 
+1. [Set up a VPC peering connection](/tidb-cloud/set-up-vpc-peering-connections.md) between the VPC of the Apache Kafka service and your TiDB cluster.
+2. Modify the ingress firewall rules of the VPC where Apache Kafka is located.
 
     You must add the CIDR of the Region where your TiDB Cloud cluster is located to the ingress firewall rules. The CIDR can be found on the VPC Peering page. Doing so allows the traffic to flow from your TiDB cluster to the Kafka brokers.
 
-### Topic
+### Kafka ACL authorization
+
+To allow TiDB Cloud changefeeds to stream data to Apache Kafka and create Kafka topics automatically, ensure that the following permissions are added in Kafka:
+
+- The `Create` and `Write` permissions are added for the topic resource type in Kafka.
+- The `DescribeConfigs` permission is added for the cluster resource type in Kafka.
+
+For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources](https://docs.confluent.io/platform/current/kafka/authorization.html#resources) and [Adding ACLs](https://docs.confluent.io/platform/current/kafka/authorization.html#adding-acls) in Confluent documentation for more information.
+
+## Step 1. Open the changefeed page for Apache Kafka
+
+1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+2. Click the name of the cluster that you want to create a changefeed for.
+3. Click the **Changefeed** tab.
+4. Click **Sink to Apache Kafka**.
+
+## Step 2. Configure the changefeed target
+
+1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
+2. Select your Kafka version. If you do not known that, use Kafka V2.
```

Comment:
```suggestion
2. Select your Kafka version. If you do not known that, use **Kafka V2**.
```

Reply:
The actual version provided on the UI is quite long. In this case, do we need to use bold text for Kafka V2?

![image](https://user-images.githubusercontent.com/79440533/187249321-c12ea841-baae-4ca3-a457-5790e2292f18.png)